chain/bitcoind_zmq_events.go

```diff
@@ -174,16 +189,85 @@ func (b *bitcoindZMQEvents) BlockNotifications() <-chan *wire.MsgBlock {
 	return b.blockNtfns
 }
 
+// getTxSpendingPrevOutReq is the rpc request format for bitcoind's
+// gettxspendingprevout call.
+type getTxSpendingPrevOutReq struct {
+	Txid string `json:"txid"`
+	Vout uint32 `json:"vout"`
+}
+
+// getTxSpendingPrevOutResp is the rpc response format for bitcoind's
+// gettxspendingprevout call. It returns the "spendingtxid" if one exists in
+// the mempool.
+type getTxSpendingPrevOutResp struct {
```

Comment:
Can also add these to `btcjson` assuming the fields are stable-ish. Also see that other issue to refactor things using tpe params so we can just do things like pass this type in and have it handle the rest of tbe boiler plate.

Reply:
which issue to refactor things? also, does this mean it would be a new `btcjson` command?

Reply:
So this issue: https://github.com/btcsuite/btcd/issues/1934

Non-blocking though.